src/imgtools/betapipeline.py

```
@@ -0,0 +1,97 @@
+        self.saver  = SampleSaver(self.output_directory)
+        self.transformer = Transformer(transforms)
+
+    def process_one_subject(self, sample):
+        images             = self.loader.load(sample)
+        transformed_images = self.transform_all(self.transforms, images)
+        metadata           = self.saver.save(transformed_images)
+        return metadata
+
+    def run(self):
+        # JSON/Dictionary of {SeriesUID: {metadata}}
+        self.db         = crawl(self.input_directory)
```

Ruff (0.8.2)

Undefined name `crawl`

(F821)